packages/compass-generative-ai/src/utils/xml-to-mms-response.ts

```diff
@@ -0,0 +1,76 @@
+import type { Logger } from '@mongodb-js/compass-logging';
+import parse, { toJSString } from 'mongodb-query-parser';
+
+export function parseXmlToMmsJsonResponse(xmlString: string, logger: Logger) {
```

Comment:
`MmsJsonResponse` feels like a technical detail to me; without context of the old assistant API I wouldn't understand what this meant.
Can we also add add a return type to this?

Reply:
added in [add type](https://github.com/mongodb-js/compass/pull/7623/commits/6eec8db3e162b15d2812d1906105ba4e0c31d5a9)